doc/howtos/booting-and-installation.rst

```diff
@@ -147,6 +147,126 @@ Flashing an Intel Edison requires use of a breakout board and two micro-USB cabl
 #. Plug in the second micro-USB cable to the J16 connector as instructed by the running flashall script.
 #. Wait for all the images to flash. You will see the progress on the flasher.
 #. Once flashing is done, the image will automatically boot up and auto-login as ``root``, no password is required.
+   
+BeagleBone Black
+================
+
+BeagleBone Black is booted from a microSD card with MBR (Master Boot Record) and not GPT (GUID Partition Table) partitions.
+Most freshly unpackaged microSD cards come with MBR partitions, but previously used ones might not.
+
+You'll probably need an adapter to use the microSD card on your host computer. If you use a microSD-to-SD adapter, 
+it will likely show up as /dev/mmcblk0. You can verify the device name assigned by using ``dmesg`` or the
+``lsblk`` command to look for the device name for the microSD card (check for a device with the size you're expecting). 
+
+In our examples below, we're using an 8GB microSD card in an SD adapter that's showing up as /dev/mmcblk0  
+(numbers and device name maybe different for your device and system).
+
+You should download the ``ostro-image-dev-beaglebone*.rootfs.tar.bz2`` image from the `Ostro Project download server`_
```

Comment:
removing zImage-am335x-bone.dtb is up to @jlaako 
As for symlinks:
```
lrwxrwxrwx 1 jenkins users        48 Mar 15 23:40 MLO -> MLO-beaglebone-v2016.01+gitAUTOINC+fa85e826c1-r0
lrwxrwxrwx 1 jenkins users        48 Mar 15 23:40 MLO-beaglebone -> MLO-beaglebone-v2016.01+gitAUTOINC+fa85e826c1-r0
-rwxr-xr-x 1 jenkins users     69076 Mar 15 23:40 MLO-beaglebone-v2016.01+gitAUTOINC+fa85e826c1-r0
lrwxrwxrwx 1 jenkins users        55 Mar 15 23:40 u-boot-beaglebone.img -> u-boot-beaglebone-v2016.01+gitAUTOINC+fa85e826c1-r0.img
-rwxr-xr-x 1 jenkins users    388048 Mar 15 23:40 u-boot-beaglebone-v2016.01+gitAUTOINC+fa85e826c1-r0.img
lrwxrwxrwx 1 jenkins users        55 Mar 15 23:40 u-boot.img -> u-boot-beaglebone-v2016.01+gitAUTOINC+fa85e826c1-r0.img
lrwxrwxrwx 1 jenkins users        76 Mar 15 23:28 zImage -> zImage--4.1.18+git0+56dcb623eb_efb6ffb2ca-r0.8-beaglebone-20160315204741.bin
-rw-r--r-- 1 jenkins users     29529 Mar 15 23:28 zImage--4.1.18+git0+56dcb623eb_efb6ffb2ca-r0.8-am335x-bone-20160315204741.dtb
-rw-r--r-- 1 jenkins users     30421 Mar 15 23:28 zImage--4.1.18+git0+56dcb623eb_efb6ffb2ca-r0.8-am335x-boneblack-20160315204741.dtb
-rw-r--r-- 1 jenkins users   5494264 Mar 15 23:28 zImage--4.1.18+git0+56dcb623eb_efb6ffb2ca-r0.8-beaglebone-20160315204741.bin
lrwxrwxrwx 1 jenkins users        82 Mar 15 23:28 zImage-am335x-boneblack.dtb -> zImage--4.1.18+git0+56dcb623eb_efb6ffb2ca-r0.8-am335x-boneblack-20160315204741.dtb
lrwxrwxrwx 1 jenkins users        77 Mar 15 23:28 zImage-am335x-bone.dtb -> zImage--4.1.18+git0+56dcb623eb_efb6ffb2ca-r0.8-am335x-bone-20160315204741.dtb
lrwxrwxrwx 1 jenkins users        76 Mar 15 23:28 zImage-beaglebone.bin -> zImage--4.1.18+git0+56dcb623eb_efb6ffb2ca-r0.8-beaglebone-20160315204741.bin
```

Those are also generated by image class for beaglebone bsp. If we want some changes there, @jlaako is maintainer for that part.